Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:8-slim
```

Comment:
I think we should rely on only one image instead of maintaining Dockerfile in every repo

Reply:
These files are unfortunately committed files. Thanks to notice!